closed/OpenJ9.gmk

```diff
@@ -470,9 +465,17 @@ ifneq (,$(CCACHE))
 endif # CCACHE
 ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
   CMAKE_ARGS += -DJITSERVER_SUPPORT=ON
+
+  ifneq (,$(OPENSSL_DIR))
+    CMAKE_ARGS += -DOPENSSL_DIR="$(OPENSSL_DIR)"
+  endif
```

Comment:
Why is `OPENSSL_CFLAGS` not propagated to the VM build here, like it is below?

Reply:
cmake calls `include_directories()` which would have the same effect as adding OPENSSL_CFLAGS to J9_CXXFLAGS and J9_CFLAGS.
https://github.com/eclipse/openj9/pull/7846#discussion_r356862010

Reply:
We should not assume that `OPENSSL_CFLAGS` only contains `-I` options.

Reply:
As per our offline discussion, I shouldn't assume `OPENSSL_CFLAGS` would contain only include path information. Will set it up in `CMAKE_ARGS` and pass it to `J9_CXXFLAGS` and `J9_CFLAGS`.